src/components/session-time-table/SessionCard.tsx

```diff
@@ -15,8 +16,10 @@ const Session = ({ session }: SessionProps) => {
 
   return (
     <div className="py-2">
-      {typeof username === "string" ? (
-        <p className="text-base font-normal text-[#040404]">{title}</p>
+      {typeof username === "string" && username ? (
+        <Link href={`/talks/${username}`} className="hover:underline">
+          <p className="text-base font-normal text-[#040404]">{title}</p>
+        </Link>
       ) : (
```

Comment:
nits: なんかtypeof使ってるのちょっと冗長な気が？？
これで良い気がするんですが何か理由あるのかな…
こっちの方が空文字も弾けるし

```suggestion
      {username ? (
        <Link href={`/talks/${username}`} className="hover:underline">
          <p className="text-base font-normal text-[#040404]">{title}</p>
        </Link>
      ) : (
```

Reply:
一部 2024 のコード持ってきた分が残ってました。修正しました。

[9d604a4](https://github.com/tskaigi/kansai-web-site/pull/38/commits/9d604a4d79518658b2a3e9a4ffbab873280d34fa)

Suggestもありがとうございます🙇‍♂️
↑のコメントと一緒に直してしまったので `Commit suggestion` できませんでした...すみません。